doc/Developer Guide/Package Publishing/Readme.md

```diff
@@ -5,12 +5,16 @@ Package Publishing
 This is a list of point to check prior to published your package on [packages.opentap.io](https://packages.opentap.io).
 
 ### Update the Package Definition (`package.xml`) file:
+* Provide a meaningful [`Name`](https://doc.opentap.io/Developer%20Guide/Plugin%20Packaging%20and%20Versioning/#attributes-in-the-configuration-file). We advise to make the names human readable, i.e. not Camel Case, no underscores, etc. No need to add "plugin" in the package name, as OpenTAP packages typically contain OpenTAP plugins. The name cannot contain '/' or be changed later, if you want to change the name you must publish the package again. 
```

Comment:
Here we say that forward slashes are not supported, but in the `Plugin Packaging And Versioning` section, we say that they are "strongly discouraged". Let's be consistent here

Reply:
This is regarding publishing, in this case a / is not supported. I would say it's the correct description